### DISCUSSION TITLE
キー反応無しのトラブルシュート

### DISCUSSION THREAD

**Discussion by boxwood8** [Yowkees/keyball#387] [Q&A] [answered]
keyball44 左手 左下の親指キー1つのみが反応しません

画像緑のラインは導通が確認できていますが、どのダイオードのアノード側と ProMicro上の端子をチェックすれば良いでしょうか
![image](https://github.com/Yowkees/keyball/assets/147312996/782d9b5e-e728-4763-b4a1-7f7b86cdae73)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

下記画像のオレンジ点どうしが導通します。
導通していないようでしたら導線を追加していただければ幸いです。

また、ProMicroの端子としては、ProMicro裏の表記で”ROW3”と書かれている端子と該当ダイオードが導通しているかご確認ください。

以上、よろしくお願いいたします。
![KB44_thumbfix](https://github.com/Yowkees/keyball/assets/84209772/81069a24-1781-4276-822b-1740ce609688)

**Reply by boxwood8** [reply]
お忙しい中早速ありがとうございました。
解消しました！
